promenade/templates/generate/validate-cluster.sh

```diff
@@ -0,0 +1,6 @@
+{% include "common_validation.sh" with context %}
+
+EXPECTED_NODE_COUNT={{ config['Cluster']['nodes'] | length }}
+wait_for_ready_nodes $EXPECTED_NODE_COUNT
+
```

Comment:
Hmm,  I was kind of hoping to delay this until we could lean on `armada test`, but we could put something in place for now and replace it with that later.